src/scripts/create_schibsted.py

```diff
@@ -55,35 +74,9 @@ def main():
 
     # Dataset is a mix of Swedish and Norwegian articles.
     # Make two separate datasets, one for each language.
-    model_path = hf_hub_download(
-        repo_id="facebook/fasttext-language-identification", filename="model.bin"
-    )
-    model = fasttext.load_model(model_path)
-
-    def detect_language(text: str) -> str:
-        """Detect if the text is in Swedish or Norwegian.
-
-        Args:
-            text:
-                The text to detect the language of.
-
-        Returns:
-            The language of the text ("sv" or "no").
-        """
-        label, _ = model.predict(text=text)
-        label = label[0]
-        if label == "__label__swe_Latn":
-            return "sv"
-        elif label == "__label__nob_Latn" or label == "__label__nno_Latn":
-            return "no"
-        return "other"
-
-    dataset_sv = dataset.filter(
-        lambda x: detect_language(text=x["article_title"]) == "sv"
-    )
-    dataset_no = dataset.filter(
-        lambda x: detect_language(text=x["article_title"]) == "no"
-    )
+    dataset_sv = dataset.filter(lambda x: x["language"] == "se")
```

Comment:
`se` is Sweden the country, not Swedish the language - the language is `sv`.

```suggestion
    dataset_sv = dataset.filter(lambda x: x["language"] == "sv")
```